.github/workflows/BACKEND_DEPLOY.yml

```
@@ -0,0 +1,79 @@
+name: 백엔드 배포
+on:
+  push:
+    branches: [main]
+    paths:
+      - "packages/backend/**"
+      - "packages/shared/**"
+jobs:
+  push_to_registry:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 코드 체크아웃
+        uses: actions/checkout@v3
+
+      - name: Docker Buildx 설정
+        uses: docker/setup-buildx-action@v2
+
+      - name: NCP 컨테이너 레지스트리 로그인
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ secrets.CONTAINER_REGISTRY_URL }}
+          username: ${{ secrets.NCP_ACCESS_KEY }}
+          password: ${{ secrets.NCP_SECERET_KEY }}
```

```suggestion
          password: ${{ secrets.NCP_SECRET_KEY}}
```

ncp 시크릿 키 오타가 있는 것 같아서 수정 제안드립니다.

---

맞습니다 🥹 fork로 CICD 테스트할 때 이것때문에 고생했던 기억이 있는데 이후에 수정되어야합니다